devtools/platform-descriptor-json/src/main/resources/bundled-codestarts/buildtool/maven/base/pom.tpl.qute.xml

```diff
@@ -12,52 +12,62 @@
         <maven.compiler.target>1.8</maven.compiler.target>
         <maven.compiler.parameters>true</maven.compiler.parameters>
 
-        <quarkus.platform.artifact-id>${bom_artifactId}</quarkus.platform.artifact-id>
-        <quarkus.platform.group-id>${bom_groupId}</quarkus.platform.group-id>
-        <quarkus.platform.version>${bom_version}</quarkus.platform.version>
-        <quarkus-plugin.version>${plugin_version}</quarkus-plugin.version>
-        <compiler-plugin.version>${compiler_plugin_version}</compiler-plugin.version>
-        <surefire-plugin.version>${surefire_plugin_version}</surefire-plugin.version>
+        <quarkus.platform.group-id>{quarkus.platform.group-id}</quarkus.platform.group-id>
+        <quarkus.platform.artifact-id>{quarkus.platform.artifact-id}</quarkus.platform.artifact-id>
+        <quarkus.platform.version>{quarkus.platform.version}</quarkus.platform.version>
+        <quarkus-plugin.version>{quarkus.plugin.version}</quarkus-plugin.version>
+        <compiler-plugin.version>{maven.version.maven-compiler-plugin}</compiler-plugin.version>
+        <surefire-plugin.version>{maven.version.maven-surefire-plugin}</surefire-plugin.version>
     </properties>
-    ${maven_repositories}${maven_plugin_repositories}
+
     <dependencyManagement>
         <dependencies>
             <dependency>
-                <groupId>${quarkus.platform.group-id}</groupId>
-                <artifactId>${quarkus.platform.artifact-id}</artifactId>
-                <version>${quarkus.platform.version}</version>
+                <groupId>$\{quarkus.platform.group-id}</groupId>
+                <artifactId>$\{quarkus.platform.artifact-id}</artifactId>
+                <version>$\{quarkus.platform.version}</version>
                 <type>pom</type>
                 <scope>import</scope>
             </dependency>
         </dependencies>
     </dependencyManagement>
 
     <dependencies>
+        {#each dependencies}
         <dependency>
-            <groupId>io.quarkus</groupId>
-            <artifactId>quarkus-resteasy</artifactId>
+            <groupId>{it.groupId}</groupId>
+            <artifactId>{it.artifactId}</artifactId>
+            {#if it.version}
+            <version>{it.version}</version>
+            {/if}
         </dependency>
+        {/each}
 
         <!-- Test dependencies -->
         <dependency>
```

Comment:
should this not be defined in base test-depends meta rather than in the template ?

Reply:
What do you mean? This is the base..

Reply:
i mean as part of what gets fed into dependencies data.